README.md

```diff
@@ -1,4 +1,4 @@
-# red5pro installer
+# Red5pro installer
 Installer for Red5 Pro Server
 
 > **NOTE**: As of Red5 Pro release 9.0, the installer can only be run on Ubuntu. If you have a downloaded server distribution version older than 9.0, then it can be run on Ubuntu 16.04 or 18.04; for version 9.0 or higher please choose Ubuntu 20.04
```

Comment:
Need to add recommendation to use ubuntu 22.04 with red5pro builds 12.0.0 and higher

Reply:
Hi @iolesyk, I have updated the README file to recommend use of `Ubuntu 22.04`.
Below is the commit URL:
https://github.com/red5pro/red5pro-installer/pull/42/commits/302cf8a7ac72ff6122de4ee9de188358a6106491